README.md

```diff
@@ -1,2 +1,37 @@
 # storcycle_api
-StorCycle API Examples
+
+[Swagger Documentation](https://localhost/apidocs/)
```

Comment:
I'd point them to https://localhost/openapi.yaml, as the link you specify, by default, shows the internal API swagger doc.  I'm not sure how to fix that.

Reply:
on every build i've tested https://localhost/apidocs/ points to https://localhost/openapi.yaml

Reply:
ex: https://10.2.2.146/apidocs/

Reply:
Good, must be unique to a dev build that it's not quite correct.